app/src/main/java/com/amaze/filemanager/filesystem/HybridFile.java

```diff
@@ -470,11 +470,9 @@ public Boolean execute(SFTPClient client) throws IOException {
                       .stat(SshClientUtils.extractRemotePathFrom(path))
                       .getType()
                       .equals(FileMode.Type.DIRECTORY);
-                } catch (SFTPException notFound) {
-                  Log.e(
-                      getClass().getSimpleName(),
-                      "Fail to execute isDirectory for SFTP path :" + path);
-                  notFound.printStackTrace();
+                } catch (IOException executionFailure) {
+                  Log.e(TAG, "Fail to execute isDirectory for SFTP path :" + path);
```

Comment:
Do `Log.e(TAG, "Fail to execute isDirectory for SFTP path :" + path, executionFailure);`